rel/overlays/bin/server

```diff
@@ -1,5 +1,5 @@
 #!/bin/sh
-
+set -e
```

Comment:
We really don't want to boot firezone if migrations or admin creation fails

Reply:
Agree